Stream_support/include/CGAL/IO/OFF/File_scanner_OFF.h

```diff
@@ -78,7 +79,7 @@ class File_scanner_OFF
     else
     {
       skip_comment();
-      std::string line;
+      line.clear();
       std::getline(m_in, line);
       // First remove the comment if there is one
       std::size_t pos = line.find('#');
```

Comment:
I just had a thought. Would it not be beneficial to do the same with the 
`std::istringstream issline(line);`

i.e move it to a class member and call `issline.clear()`

Reply:
you can set the content with `issline.str(line)`